
class ConnectionError(Exception):
    """Ошибка соединения."""

    pass
